internal/deviations/deviations.go

```diff
@@ -1135,3 +1135,7 @@ func DecapNHWithNextHopNIUnsupported(dut *ondatra.DUTDevice) bool {
 func SflowSourceAddressUpdateUnsupported(dut *ondatra.DUTDevice) bool {
 	return lookupDUTDeviations(dut).GetSflowSourceAddressUpdateUnsupported()
 }
+
+func ChassisGetRpcUnsupported(dut *ondatra.DUTDevice) bool {
```

Comment:
Fix static check:

exported function ChassisGetRpcUnsupported should have comment or be unexported